wpilibc/Athena/include/DriverStation.h

```diff
@@ -104,17 +105,24 @@ class DriverStation : public SensorBase, public RobotStateInterface {
   void ReportJoystickUnpluggedWarning(std::string message);
   void Run();
 
-  HALJoystickAxes m_joystickAxes[kJoystickPorts];
-  HALJoystickPOVs m_joystickPOVs[kJoystickPorts];
-  HALJoystickButtons m_joystickButtons[kJoystickPorts];
-  HALJoystickDescriptor m_joystickDescriptor[kJoystickPorts];
+  std::unique_ptr<HALJoystickAxes[]> m_joystickAxes;
+  std::unique_ptr<HALJoystickPOVs[]> m_joystickPOVs;
+  std::unique_ptr<HALJoystickButtons[]> m_joystickButtons;
+  std::unique_ptr<HALJoystickDescriptor[]> m_joystickDescriptor;
+  // Cached Data
```

Comment:
I would prefer a newline before this comment to separate the two groups of variables.